src/Module.cpp

```diff
@@ -10,114 +17,225 @@ Module::Module(std::string moduleName, WContainerWidget *parent)
   const std::string MENUDATA_XML_FILE = Wt::WApplication::instance()->docRoot() + "/xml/"+ moduleName +".xml";
   pugi::xml_parse_result tos = doc.load_file(MENUDATA_XML_FILE.c_str());
 
-  //  std::cout << "Load result: " << tos.description() << std::endl;
-
-  pugi::xml_node task_node = doc.child("task");
-  pugi::xml_node taskdescription_node = task_node.child("description");
-  pugi::xml_node para_node = task_node.child("parameter");
-  std::vector<std::string> name;
-  std::vector<std::string> str1;
-  int i=0;
-  while(para_node){
-    /*first iterating over the required box*/
-    std::string jj = para_node.attribute("required").value();
-                        
-    if(jj=="yes")
-      { 
-	pugi::xml_node require_node = para_node.child("label"); /*if label is available*/
-                       
-	std::string nu=require_node.child_value();
-
-	if(nu.empty())
-	  { pugi::xml_node require_node = para_node.child("description"); /*else description*/
-	    str1.push_back(require_node.child_value());
-	    // std::cout<<require_node.child_value()<<std::endl;
-	    // std::cout<<str1[i]<<std::endl;
-	    i++;
-	  }
-	else{
-	  str1.push_back(require_node.child_value());
-	  // std::cout<<require_node.child_value()<<std::endl;
-	  // std::cout<<str1[i]<<std::endl;
-	  i++;
-	}
+  std::cout << "Load result: " << tos.description() << std::endl;
+  std::string check= tos.description();
+  if(check=="File was not found")
+   {
+      // JavascriptExample::setTitle("Javascript example");
+      // promptAmount_ = Popup::createPrompt("How much do you want to pay?", "",this);
+
+      Wt::WContainerWidget *error = new Wt::WContainerWidget();
+      Wt::WText *text1 = new Wt::WText("File was not found", error);
+      addWidget(error);
```

Comment:
what is the exact issue with changing to doJavaScript()
https://www.webtoolkit.eu/wt/doc/reference/html/classWt_1_1WApplication.html#a2a92457b9212cef4057cb54e56183967

simple google search gives good results.  whatever the issue is reply on the gsoc report thread